src/components/interfaces/MOBILE_API.xml

```diff
@@ -7489,6 +7493,7 @@
         <param name="permissionItem" type="PermissionItem" minsize="0" maxsize="500" array="true" mandatory="true">
             <description>Change in permissions for a given set of RPCs</description>
         </param>
+        <param name="requireEncryption" type="Boolean" mandatory="false" since="5.1"/>
```

Comment:
```suggestion
        <param name="requireEncryption" type="Boolean" mandatory="false" since="6.0"/>
```

Reply:
fixed in 45f0888b531adde8ac91715a3c526c75c7c23ba3